README.md

```diff
@@ -36,7 +36,11 @@ To install `poutine`, download the latest release from the [releases page](https
 
 #### Homebrew
 ``` bash
-brew install boostsecurityio/tap/poutine
+[~] brew verify poutine
+==> poutine--0.10.1.sonoma.bottle.tar.gz has a valid attestation
+
+[~] brew install poutine
```

Comment:
just include the command it makes it easier to copy-paste, strip the other output.